packages/app/src/AppNav.tsx

```diff
@@ -604,7 +604,10 @@ export default function AppNav({ fixed = false }: { fixed?: boolean }) {
         }}
       >
         <div style={{ width: navWidth }}>
-          <div className="p-3 d-flex flex-wrap justify-content-between align-items-center">
+          <div
+            className="p-3 d-flex flex-wrap justify-content-between align-items-center"
+            style={{ height: 58 }}
+          >
```

Comment:
With this change and the one in packages/app/src/AppNav.components.tsx, the sidebar icons overlap with the expand/collapse button when the navbar is collapsed:

<img width="57" height="355" alt="Screenshot 2025-11-19 at 2 04 42 PM" src="https://github.com/user-attachments/assets/5dd3c57b-5efa-44c9-a8fc-e1f6d55b1eae" />